luda-editor/luda-editor-client/src/app/editor/clip_select.rs

```diff
@@ -61,10 +64,15 @@ impl Editor {
 
         selected_clip_track.get_id() == clip_track.get_id()
     }
-    pub(super) fn select_all_between_clips<T: AsRef<str>>(&mut self, clip_ids: &[T]) {
+    pub(super) fn select_all_to_time(&mut self, time: &Time) {
```

Comment:
We only think about adding clips. Later, I gonna put the selecting functionality like a text cursor or windows explorer, which support to remove clips from selected clips.